keyboards/kbdfans/kbd67/rev2/rev2.h

```diff
@@ -145,4 +145,19 @@
     { K40, K41,   KC_NO, K43, K44, KC_NO, K46, KC_NO, K48, KC_NO, K4A, K4B, K4C,   K4D, K4E,   K4F }, \
 }
 
+#define LAYOUT_65_ansi_split_bs_split_space( \
```

Comment:
Typically denoting `split` in a layout macro within QMK is all-encompassing for whichever keys are listed afterward (`split_bs_space` meaning both Backspace and Space are split).

```suggestion
#define LAYOUT_65_ansi_split_bs_space( \
```